examples/Polls API.md

```diff
@@ -0,0 +1,179 @@
+                    "url": "/questions/1",
+                    "choices": [
+                        {
+                            "answer": "Swift",
+                            "url": "/questions/1/choices/1",
+                            "votes": 2048
+                        }, {
+                            "answer": "Python",
+                            "url": "/questions/1/choices/2",
+                            "votes": 1024
+                        }, {
+                            "answer": "Objective-C",
+                            "url": "/questions/1/choices/3",
+                            "votes": 512
+                        }, {
+                            "answer": "Ruby",
+                            "url": "/questions/1/choices/4",
+                            "votes": 256
+                        }
+                    ]
+                }
+
+## Choice [/questions/{question_id}/choices/{choice_id}]
+
++ Parameters
+
+    + question_id (number) ... ID of the Question in form of an integer
+    + choice_id (number) ... ID of the Choice in form of an integer
+
+### Vote on a Choice [PUT]
+
+This action allows you to vote on a question's choice.
+
++ Response 201
+
+    + Headers
+
+            Location: /questions/1
+
+## Questions collection [/questions{?page}]
+
+Again, instead of constructing the URLs for the next page. It is **highly** recommended that you follow the `next` link header in the response.
+
++ Parameters
+
+    + page (optional, number) ... The page of questions to return
+
+### List all questions [GET]
+
++ Response 200 (application/json)
+
+    + Headers
+    
+            Link: </questions?page=2>; rel="next"
+
+    + Body
+
+            [
+                {
+                    "question": "Favourite programming language?",
+                    "published_at": "2014-11-11T08:40:51.620Z",
+                    "url": "/questions/1",
+                    "choices": [
+                        {
+                            "answer": "Swift",
+                            "url": "/questions/1/choices/1",
+                            "votes": 2048
+                        }, {
+                            "answer": "Python",
+                            "url": "/questions/1/choices/2",
+                            "votes": 1024
+                        }, {
+                            "answer": "Objective-C",
+                            "url": "/questions/1/choices/3",
+                            "votes": 512
+                        }, {
+                            "answer": "Ruby",
+                            "url": "/questions/1/choices/4",
+                            "votes": 256
+                        }
+                    ]
+                }
+            ]
+
+### Create a new question [POST]
+
+You can create your own question using this action. It takes a JSON dictionary containing a question and a collection of answers in the form of choices.
+
+- question (string) - The question
```

Comment:
The other tutorials such as Gist Fox also are doing this. Personally I have been using the `+` for API Blueprint sections such as `Response` `Parameters` etc. But used `-` for other lists.

I'm happy to change these to `+` along with the other tutorials if you would rather them all use `+`.